### DISCUSSION TITLE
Wireguard Logs - Crowdsec

### DISCUSSION THREAD

**Discussion by trentnbauer** [wg-easy/wg-easy#728] [Ideas]
Hi all,

Where does WG-Easy place its Wireguard logs?
I'm rolling out Crowdsec and want to add some protection in front of Wireguard.
Crowdsec doco here: https://app.crowdsec.net/hub/author/crowdsecurity/collections/wireguard

> In order for CrowdSec to detect attacks on Wireguard it needs logs and since Wireguard by default logs close to nothing we need to enable Wireguard's dyndbg logging which sends log messages to the Linux kernel message buffer, kmsg. These will be picked up by your Linux distro's syslog service (at least on Debian, probably also on most others) and logged in /var/kern.log. On other distros they will be logged to /var/log/messages.
```
sudo modprobe wireguard
echo module wireguard +p | sudo tee /sys/kernel/debug/dynamic_debug/control
```

**Comment by pheiduck** [comment]
WG-Easy runs in a Docker container maybe you need to monitor the traffic from the Docker container.
But currently I have no knowledge about Crowdsec.
And at the next time please use [Discussions](https://github.com/wg-easy/wg-easy/discussions) for questions like this. 😉
